ClearKeep/Scenes/InCall/VideoContainerView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  VideoContainerView.swift
+//  ClearKeep
+//
+//  Created by Luong Minh Hiep on 19/05/2021.
+//
+
+import SwiftUI
+import CommonUI
+import ChatSecure
+
+struct VideoContainerView: View {
+    @ObservedObject var viewModel: CallViewModel
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/InCall/VideoContainerView.swift#L13"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>VideoContainerView.swift:13</code></td>
    </tr>
  </tbody>
</table>